scripts/install_third_party.py

```diff
@@ -128,9 +129,9 @@ def download_and_untar_files(
         print 'Downloading and untarring file %s to %s' % (
             tar_root_name, target_parent_dir)
         common.ensure_directory_exists(target_parent_dir)
-
         urllib.urlretrieve(source_url, TMP_UNZIP_PATH)
-        with tarfile.open(TMP_UNZIP_PATH, 'r:gz') as t:
+	print TMP_UNZIP_PATH
+        with closing(tarfile.open(TMP_UNZIP_PATH, 'r:gz')) as t:
```

Comment:
I like this (well done on finding it!) -- but I'm also confused. Why did this trigger just for numpy, and not for the existing third-party libs?